scrapers/Andomark.yml

```diff
@@ -34,9 +74,73 @@ xPathScrapers:
             selector: //link[@rel="canonical"]/@href
             postProcess:
                 - replace:
-                    - regex: (.+\/\/)(.+?)(\..+)
+                    - regex: .elscomplete
+                      with: "" 
```

Comment:
Not sure if this is necessary, is this for `reidmylips.elxcomplete.com`? The next step will only grab the first part of the domain name either way :)
```suggestion
```